msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -282,6 +284,96 @@ The following table includes the response codes:
 | **403** | The app is unable to send the signal. 403 response code can occur because of various reasons, such as the tenant admin disables and blocks the app during live site migration. In this case, the payload contains a detailed error message. |
 | **404** | The meeting chat doesn't exist. |
 
+## Targeted in-meeting notification API
+
+The `targetedMeetingNotification` API allows apps to send notifications to specific participants in a meeting. Apps send targeted in-meeting notifications based on user action. The API is available through bot API.
+
+> [!NOTE]
+>
+> * The API payload only permits a task module with a URL.
+> * The user ID formats **aadObjectid** and **UPN** aren't supported.
+
+Get supported user ID format for targeted in-meeting notification:
+
+* [Get participant API](#get-participant-api)
+* [Get members API](../bots/how-to/get-teams-context.md#fetch-the-roster-or-user-profile)
+
+### Example
+
+Following is an example of request payload:
```

Comment:
Can we update the statement like this? 
"Following is an example of request payload for targeted in-meeting notification feature. "

The reason is in the future we are going to support a different type of surface, and the payload will look different.

Reply:
Updated.